hazelcast/src/main/java/com/hazelcast/spi/impl/operationservice/impl/OperationRunnerImpl.java

```diff
@@ -239,7 +240,7 @@ protected void ensureNoPartitionProblems(Operation op) {
     }
 
     private boolean retryDuringMigration(Operation op) {
-        return !(op instanceof ReadonlyOperation || OperationAccessor.isMigrationOperation(op));
+        return !(op instanceof ReadonlyOperation || Operations.isMigrationOperation(op));
```

Comment:
Can you use a static import. So the Operations. stuff isn't needed anymore. I'm trying to apply them where possible to make code easier to read (less long lines).